doc/nrf/app_dev/optimizing/power.rst

```diff
@@ -86,17 +86,75 @@ Cellular applications
 An nRF9160 DK can draw current ranging from a few micro amperes (in sleep mode) to hundreds of milli amperes (when the radio is active).
 To achieve long battery life, it is crucial that the application is optimized in the use of the radio.
 
+PSM usage
+=========
+
+Power Saving Mode (PSM) enables the modem to enter deep sleep when it does not need to communicate with the network.
+The modem can spend most of the time in deep sleep, only waking up when data needs to be sent and received.
+When implementing an application taking advantage of the PSM, keep in mind that the enabling PSM request does not mean that modem can always enter PSM.
+
+Possible cases when PSM can not be used:
+
+* Network does not support PSM.
+* Roaming network supports PSM, but does not allow the roaming subscription to use it.
+* Modem does not find a suitable cell.
+
+You can detect the first two cases by checking the ``+CEREG`` notification or ``AT%XMONITOR`` command output for Active-Time granted by the network.
+If the Active-Time is ``11100000``, the timer is deactivated and PSM can not be used.
+Depending on the use case, it may be preferable to set modem to offline mode during long periods of inactivity to conserve power when PSM is not available.
+
+The third case can happen when cellular network coverage is poor.
+After modem has entered PSM, it stays in deep sleep regardless of cellular network availability.
+When modem wakes up from PSM to transmit data or to do a periodic Tracking Area Update (TAU), it starts searching for a suitable cell.
+If a suitable cell is not found during the initial search, modem will continue periodic cell searches until a suitable cell is found.
+Because modem can only enter PSM when it’s in registered state and camped on suitable cell, the modem can not enter PSM until a suitable cell has been found.
```

Comment:
```suggestion
After the modem has entered PSM, it stays in deep sleep regardless of cellular network availability.
When the modem wakes up from PSM to transmit data or to do a periodic Tracking Area Update (TAU), it starts searching for a suitable cell.
If a suitable cell is not found during the initial search, the modem will continue periodic cell searches until a suitable cell is found.
Because modem can only enter PSM when it’s in registered state and camped on suitable cell, the modem can not enter PSM until a suitable cell has been found.
```